mne/epochs.py

```diff
@@ -825,16 +831,26 @@ def average(self, picks=None):
 
         Notes
         -----
-        Computes an average of all epochs in the instance, even if
+        Computes an aggregate of all epochs in the instance, even if
```

Comment:
I am a bit concerned to call average a function that can do more types of aggregations.

I would keep average as it is and add an .agg method in Epochs following the Pandas naming.

Reply:
One motivation to keep it the same would be that it gets stored in FIF as type "average" (the only other choice I know if is "standard deviation"), so we do mirror the file format properly. Adding an `agg` method will not. This PR just allows you to set the the type of "average"-like operation you use (mean, median, trimmed mean, median-of-means, etc.) that will be called "average" in the file you save.

Reply:
@agramfort does that convince you?

Reply:
keep average